packages/overlay/src/integrations/sentry/components/insights/agents/AITraceItem.tsx

```diff
@@ -0,0 +1,40 @@
+import type { SpotlightAITrace } from "~/integrations/sentry/types";
+import { getFormattedDuration } from "~/integrations/sentry/utils/duration";
+import DateTime from "../../shared/DateTime";
+interface AITraceItemProps {
+  trace: SpotlightAITrace;
+  onClick: () => void;
+}
+
+export default function AITraceItem({ trace, onClick }: AITraceItemProps) {
+  if (!trace) {
+    return null;
+  }
+
+  const { id, name, operation, timestamp, durationMs, tokensDisplay } = trace;
+
+  return (
+    <tr
+      className="hover:bg-primary-900 group cursor-pointer"
+      onClick={onClick}
+      role="link"
+      tabIndex={0}
+      onKeyDown={e => {
+        if (e.key === "Enter" || e.key === " ") {
+          onClick();
+        }
```

Comment:
I think @Shubhdeep12 has somewhat a standardized handler for this? If not we should definitely invest in this.